### PR TITLE
Fix helm3 validation issue on credentiald

### PR DESCRIPTION
**Pull request by tomahawk28**
Fixing error from helm3 validation. 

```
kg get app credentiald-unique -o yaml | yq .status.release
{
  "lastDeployed": "2020-06-29T12:35:17Z",
  "reason": "Helm reason: Upgrade complete Operator reason: helm validation error: (error validating \"\": error validating data: unknown object type \"nil\" in Secret.data.aws.admin.arn)",
  "status": "validation-failed"
}
```

### CODE REVIEW

File: helm/credentiald/Chart.yaml [hunk 1]
```diff
@@ -3,5 +3,5 @@ name: credentiald
 description: >-
   credentiald manages credentials for cloud environments.
 home: https://github.com/giantswarm/credentiald
-version: [[ .Version ]]
-appVersion: [[ .AppVersion ]]
+version: 100.0.0
+appVersion: 100.0.0
```
Comment (rossf7) [inline helm/credentiald/Chart.yaml:7]: The test code needs to be removed. Also this architect templating is so annoying.

File: helm/credentiald/templates/credential-default.yaml [hunk 2]
```diff
@@ -12,13 +12,13 @@ metadata:
     giantswarm.io/service-type: system
 data:
   {{- if .Values.Installation.V1.Secret.Credentiald.AWS }}
-  aws.admin.arn: {{ .Values.Installation.V1.Secret.Credentiald.AWS.CredentialDefault.AdminARN | b64enc }}
-  aws.awsoperator.arn: {{ .Values.Installation.V1.Secret.Credentiald.AWS.CredentialDefault.AWSOperatorARN | b64enc }}
+  aws.admin.arn: {{ .Values.Installation.V1.Secret.Credentiald.AWS.CredentialDefault.AdminARN | b64enc | quote }}
```
Comment (rossf7) [inline helm/credentiald/templates/credential-default.yaml:15]: Nice, good catch! 👍

File: helm/credentiald/Chart.yaml (hunk 1 above)
Comment (tomahawk28) [reply helm/credentiald/Chart.yaml:7]: Me either, Can we maybe escalate this to team-biscuit ?

Comment (rossf7) [reply helm/credentiald/Chart.yaml]: Yes will do. There is also the idea to have a VERSION file and not need this templating.

https://gigantic.slack.com/archives/C04TGHDEF/p1591257038015600

I'll also check how that is going.

**Pull request merged**